hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -592,10 +592,20 @@ else:
       .alert_rx_i  ( alert_rx[${slice}] ),
     % endif
     ## TODO: Inter-module Connection
-    % if m["type"] == "flash_ctrl":
-
-      .flash_o(flash_req),
-      .flash_i(flash_rsp),
+    % if "inter_signal_list" in m:
```

Comment:
this just uses `inter_signal_list`. what is the expectation for usage of `inter_module`?

Reply:
It was temporary. I've revised to use `inter_module` (or `inter_signal`) to define the signal at the top.